py3status/modules/sql.py

```diff
@@ -18,9 +18,9 @@
     Parameters can be placeholders too, eg {host}, {passd}
 
 Format_row placeholders:
-    {xxx} case-sensitive placeholder name often found in the database
-    {xxx} case-sensitive placeholder name often found in the database
-    {xxx} case-sensitive placeholder name often found in the database
+    {xxx} placeholder name often found in the database
+    {xxx} placeholder name often found in the database
+    {xxx} placeholder name often found in the database
```

Comment:
`{xxx} placeholder name often found in the database`??
 something like
`{field_name} placeholders will have the value returned by the query for the field`

Reply:
`{field_name}` could  be misinterpreted. I do `{xxx}` porn w/ your statement.

Reply:
@tobes Are we good here to merge?

Reply:
You drive a hard bargain. Pushedxxx.